NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs

```diff
@@ -63,16 +66,16 @@ protected override void OnCreate(Bundle savedInstanceState)
             _switchCustom = FindViewById<SwitchCompat>(Resource.Id.welcome_page_five_switch);
             _switchCustom.CheckedChange += OnCheckedChange;
             _switchCustom.ContentDescription = ConsentViewModel.SWITCH_ACCESSIBILITY_CONSENT_SWITCH_DESCRIPTOR;
+            _switchCustom.TextOn = ConsentViewModel.GIVE_CONSENT_TEXT;
+            _switchCustom.TextOff = ConsentViewModel.GIVE_CONSENT_TEXT;
+            SetSwitchButtonTextColor();
+            SetUrduSwitchButton(LocalPreferencesHelper.GetAppLanguage());
```

Comment:
Miss-leading function name I assume? Why only Urdu is relevant?

Reply:
Urdu translation is longer than the others so had to split it into to lines instead one to be able to support large text - so SetUrduSwitchButton changes the design of the switch button to be higher, such that it can contain 2 lines instead of one.

Reply:
What about larger texts? Did you test out the feature with setting text to larger size for other languages? Maybe the problem will appear for them as well.

Reply:
Have tested with larger text, by increasing phone font size, on all languages it was only a problem with Urdu

Reply:
Pls rename function to smth like:
"SetSwitchButtonForLongString". Have an array in it containing all the languages with long string for switch button (for now it is only urdu, but in future there might be more). Thus, it will be clear for devs in the future how to implement support for longtext if they will run into the same issue.